script/appsignal

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cd "$(dirname "$0")/.."
+
+if [[ $DYNO == $APPSIGNAL_DYNO_ENABLED ]]; then
+    bundle exec appsignal install
```

Comment:
What happens when we spin up a new dyno? Is there some other way of provisioning that I am missing? (Ideally this should be a part of a provisioning script.)